src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http3.cs

```diff
@@ -1438,6 +1438,7 @@ public async Task DuplexStreaming_AbortByServer_StreamingCancelled(bool graceful
             new TheoryData<string>
             {
                 { "https://cloudflare-quic.com/" }, // Cloudflare with content
+                { "https://quic.nginx.org/" }, // Nginx with content
                 // This endpoint is consistently failing.
                 // [ActiveIssue("https://github.com/dotnet/runtime/issues/63009")]
                 //{ "https://pgjones.dev/" }, // aioquic with content
```

Comment:
I would probably remove this and close #63009 since we now have more than one server again.

Reply:
Done